### PR TITLE
Add stringify-object declaration

### PR DESCRIPTION
**Pull request by pascalduez**
See https://github.com/yeoman/stringify-object

### CODE REVIEW

File: definitions/npm/stringify-object_v3.x.x/test_stringify-object_3.x.x.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+// @flow
```
Comment (AndrewSouthpaw) [inline definitions/npm/stringify-object_v3.x.x/test_stringify-object_3.x.x.js]: Thanks for writing tests; please organize them with `describe` and `it` blocks.

File: definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+declare module 'stringify-object' {
+  declare type AnyObject = { [key: mixed]: mixed };
```
Comment (AndrewSouthpaw) [inline definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js]: Any reason to use the indexer property here? I think `{}` would suffice.

Comment (pascalduez) [reply definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js]: That's what I got first but the `obj[prop]` gives error.

```
Cannot access computed property using mixed [1].

      test_stringify-object_3.x.x.js:64:37
      64|   filter: (obj, prop) => typeof obj[prop] === 'string',
                                              ^^^^

   References:
      stringify-object_v3.x.x.js:7:30
       7|     filter?: (obj: {}, prop: mixed) => boolean,
                                       ^^^^^ [1]
```

Happy to get insights on this.

File: definitions/npm/stringify-object_v3.x.x/test_stringify-object_3.x.x.js (hunk 1 above)
Comment (pascalduez) [reply definitions/npm/stringify-object_v3.x.x/test_stringify-object_3.x.x.js]: Done.

File: definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js (hunk 2 above)
Comment (pascalduez) [reply definitions/npm/stringify-object_v3.x.x/flow_v0.50.x-/stringify-object_v3.x.x.js]: Done.

**Pull request merged**